lighthouse-core/scripts/timings.js

```diff
@@ -0,0 +1,139 @@
+  .array('urls')
+  .string('lh-flags')
+  .default('lh-flags', '')
+  // Why is the printing for examples so awful?
+  // eslint-disable max-len
+  // .example('node lighthouse-core/scripts/timings.js --name my-collection --collect -n 3 --lh-flags=\'--only-audits=unminified-javascript\' --urls https://www.example.com', 'Collect')
+  // .example('node lighthouse-core/scripts/timings.js --name my-collection --summarize --measure-filter \'loadPage|connect\'', 'Summarize')
+  // eslint-enable max-len
+  .wrap(yargs.terminalWidth())
+  .argv;
+
+const outputDir = `${ROOT_OUTPUT_DIR}/${argv.name}`;
+
+if (argv.collect) {
+  if (!fs.existsSync(ROOT_OUTPUT_DIR)) fs.mkdirSync(ROOT_OUTPUT_DIR);
+  if (fs.existsSync(outputDir)) throw new Error(`folder already exists: ${outputDir}`);
+  fs.mkdirSync(outputDir);
+
+  for (const url of argv.urls) {
+    for (let i = 0; i < argv.n; i++) {
+      const cmd = [
+        'node',
+        `${LH_ROOT}/lighthouse-cli`,
+        url,
+        `--output-path=${outputDir}/lhr-${url.replace(/[^a-zA-Z0-9]/g, '_')}-${i}.json`,
+        '--output=json',
+        argv.lhFlags,
+      ].join(' ');
+      execSync(cmd, {stdio: 'ignore'});
+    }
+  }
+}
+
+/**
+ * @param {number[]} values
+ */
+function average(values) {
+  return values.reduce((sum, value) => sum + value) / values.length;
+}
+
+/**
+ * Round to the tenth.
+ * @param {number} value
+ */
+function round(value) {
+  return Math.round(value * 10) / 10;
+}
+
+if (argv.summarize) {
+  /** @type {Map<string, number[]>} */
+  const measuresMap = new Map();
+  /** @type {RegExp|null} */
+  const measureFilter = argv.measureFilter ? new RegExp(argv.measureFilter, 'i') : null;
+
+  for (const lhrPath of fs.readdirSync(outputDir)) {
+    const lhrJson = fs.readFileSync(`${outputDir}/${lhrPath}`, 'utf-8');
+    /** @type {LH.Result} */
+    const lhr = JSON.parse(lhrJson);
+
+    for (const measureName of lhr.timing.entries.map(entry => entry.name)) {
+      if (measureFilter && !measureFilter.test(measureName)) {
+        continue;
+      }
+
+      const measuresKey = `${lhr.requestedUrl}@@@${measureName}`;
+      let measures = measuresMap.get(measuresKey);
+      if (!measures) {
+        measures = [];
+        measuresMap.set(measuresKey, measures);
+      }
+
+      const measureEntry = lhr.timing.entries.find(measure => measure.name === measureName);
+      if (!measureEntry) throw new Error('missing measure');
+
+      measures.push(measureEntry.duration);
+    }
+  }
+
+  const results = [...measuresMap.entries()].map(([measuresKey, measures]) => {
+    const [url, measureName] = measuresKey.split('@@@');
+    const mean = average(measures);
+    const min = Math.min(...measures);
+    const max = Math.max(...measures);
+    const stdev = Math.sqrt(average(measures.map(measure => (measure - mean) ** 2)));
+    return {
+      measure: measureName,
+      url,
+      n: measures.length,
+      mean: round(mean),
+      stdev: round(stdev),
+      min,
```

Comment:
round these bad boys too? :)

Reply:
they actually already get truncated to hundredths (https://github.com/GoogleChrome/lighthouse/blob/266404d2b2d85c2e5ad7a770a12ab0adf7cf7830/lighthouse-core/runner.js#L190) so I didn't notice, but yeah for consistency will round these too